app/jobs/autotune/sync_blueprint_job.rb

```diff
@@ -10,7 +10,8 @@ class SyncBlueprintJob < ActiveJob::Base
     end
 
     # do the deed
-    def perform(blueprint, status: nil, update: false)
+    # def perform(blueprint, project, status: nil, update: false)
+    def perform(blueprint, status: nil, update: false, project: false)
```

Comment:
Don't pass a project object, change this to `build_preview` and pass in true or false.